### PR TITLE
Add Disaster recovery

### PR DESCRIPTION
**Pull request by r0ckarong**
Accidentally merged and reverted #698, bring back changes.

### CODE REVIEW

**Comment by r0ckarong** [comment]
Currentlly conflicts with the changes introduced with the KVM support. I will resolve these once KVM is shipped and we are actually working on the release that includes Velero.

**Comment by innobead** [comment]
> Currentlly conflicts with the changes introduced with the KVM support. I will resolve these once KVM is shipped and we are actually working on the release that includes Velero.

super and thanks :+1:

File: adoc/admin-dr-prereqs.adoc [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
++
+[source,bash]
+----
+aws s3api create-bucket \
+    --bucket <BUCKET_NAME> \
+    --region <REGION> \
+    --create-bucket-configuration LocationConstraint=<REGION>
+----
+
+. Prepare the credential file `credentials-velero` in the local machine
```
Comment (c3y1huang) [inline adoc/admin-dr-prereqs.adoc]:
```suggestion
. Create the credential file `credentials-velero` in the local machine
```
nit: align with other provider instructions.

File: adoc/admin-dr-prereqs.adoc [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+
+# Bind iam policy to project
+gcloud projects add-iam-policy-binding $PROJECT_ID \
+    --member serviceAccount:$SERVICE_ACCOUNT_EMAIL \
+    --role projects/$PROJECT_ID/roles/velero.server
+
+gsutil iam ch serviceAccount:$SERVICE_ACCOUNT_EMAIL:objectAdmin gs://<BUCKET_NAME>
+----
+
+. Create a credential file `credentials-velero` in the local machine
```
Comment (c3y1huang) [inline adoc/admin-dr-prereqs.adoc]:
```suggestion
. Create the credential file `credentials-velero` in the local machine
```
nit: align with other provider instructions.

File: adoc/admin-dr-prereqs.adoc [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
++
+Create a blob container named velero. Change the name as needed.
++
+[source,bash]
+----
+BLOB_CONTAINER=velero
+az storage container create -n $BLOB_CONTAINER --public-access off --account-name $AZURE_STORAGE_ACCOUNT_ID
+----
+
+. Create a credential file `credentials-velero` in the local machine
```
Comment (jenting) [inline adoc/admin-dr-prereqs.adoc]:
```suggestion
. Create the credential file `credentials-velero` in the local machine
```

**Comment by jordimassaguerpla** [comment]
Velero is ready to be released https://build.suse.de/request/show/214429. You can proceed with merging this.

**Pull request merged**